op-guide/backup-restore.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 备份恢复
+category: advanced
+---
+
+# 备份恢复
+
+## 概述
+
+
+该文档详细介绍了如何对 TiDB 进行备份恢复。本文档暂时只考虑全量备份与恢复。
```

Comment:
Looks like MySQL has incremental backup and restore...
https://dev.mysql.com/doc/refman/5.7/en/point-in-time-recovery.html